src/Features/Cart/CartProductList.js

```diff
@@ -1,30 +1,45 @@
-import React from 'react'
-import { keys } from 'lodash'
-import products, { getProductsMap } from '../../App/Main/Product/products'
-import CartProductListItem from './CartProductListItem'
+import { keys } from 'lodash';
+import PropTypes from 'prop-types';
+
+import products, { getProductsMap } from '../../App/Main/Product/products';
+import CartProductListItem from './CartProductListItem';
 
 const CartProductList = ({
-    productsInCart,
-    productMap = getProductsMap(products),
-    ListItemComponent = CartProductListItem,
-    removeProductFromCart,
-    setProductQuantityInCart,
+  productsInCart,
+  productMap = getProductsMap(products),
+  ListItemComponent = CartProductListItem,
```

Comment:
Do we need to pass this component via props? Can we just import `CartProductListItem`